packages/interbit-ui-components/src/tests/uiKit.test.js

```diff
@@ -328,3 +329,41 @@ describe('<TitledList />', () => {
     countChildren(<TitledList {...props} />, 'li', props.items.length)
   })
 })
+
+describe('<ActivityBar />', () => {
+  const props = {}
+  beforeEach(() => {
+    props.firstName = 'foo'
+    props.secondName = 'bar'
+    props.breadcrumb = [{ title: 'foo' }, { title: 'bar' }]
+    props.userClickHandler = jest.fn()
+    props.timestamp = 'Tue Sep 25 18:19:57 2018'
```

Comment:
Can you change the timestamp to be in milliseconds from the Unix epoch? See the `value` parameter in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date. 

Your code is fine, but we will be passing in timestamps in the above format so the test should reflect that.